test/views/wallet/ConfigureMobilecoindView/LedgerStatus.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+
+import useMobileCoinD from '../../../../app/hooks/useMobileCoinD';
+import LedgerStatus from '../../../../app/views/wallet/ConfigureMobilecoindView';
+
+jest.mock('../../../../app/hooks/useMobileCoinD');
+
+describe('LedgerStatus View', () => {
+  test('Leddger Staus view renders along with block types', () => {
+    const mockUseMobileCoinD = useMobileCoinD as jest.MockedFunction<
+      typeof useMobileCoinD
+    >;
+
+    // @ts-ignore mock
+    mockUseMobileCoinD.mockImplementation(() => {
+      return {
+        accountName: 'accountName',
+        b58Code: 'string',
+        balance: BigInt(2),
+        encryptedEntropy: 'string',
+        entropy: null,
+        giftCodes: null,
+        isAuthenticated: true,
+        isEntropyKnown: true,
+        isInitialised: true,
+        localBlockIndex: 4,
+        mobUrl: 'string',
+        monitorId: null,
+        networkHighestBlockIndex: 4,
+        nextBlock: 5,
+        receiver: null,
+      };
+    });
+
+    const history = createMemoryHistory();
+    render(
+      <Router history={history}>
+        <LedgerStatus />
+      </Router>
+    );
```

Comment:
Now that we have `renderSnapshot`, let's use that instead. I typically prefer creating a setup method to be called inside the test to prevent too much bloat in the test itself.